<launch>
	
	<include file="$(find daniels)/launch/includes/minimal.launch.xml"/>
	
	<include file="$(find daniels)/launch/includes/sick_tim571_2050101.launch.xml"/>
<!--
	<node pkg="daniels" name="ScanBall" type="ScanBall" output="screen" />
-->
	<include file="$(find daniels)/launch/includes/amcl.launch.xml"/>
	
	<include file="$(find daniels)/launch/includes/move_base.launch.xml"/>
<!--
	<node pkg="daniels" name="Player" type="player.py" output="screen" />
-->
</launch>
